### ISSUE TITLE
Issue: readthedocs integration no longer working correctly for PRs

### ISSUE THREAD

**Issue by climbfuji** [JCSDA/spack-stack#601]
**Describe the bug**
PRs still trigger the readthedocs build, but the integration on the PR page is missing.

**To Reproduce**
Check any PR.

**Expected behavior**
PR shows the link to the readthedocs page for this PR as part of the tests to run

**System:**
n/a

**Additional context**
n/a

**Comment by climbfuji** [comment]
Dom to check the readthedocs configuration file in the spack-stack repo.

**Comment by climbfuji** [comment]
Could also be related to our github organizations (don't see the hookups in jedi-docs either).

**Comment by climbfuji** [comment]
I spent too much time on finding out why this happens. My suspicion is that it has to do with the JCSDA organization being "different" from the other orgs, because it's not showing up in the list of my organizations in readthedocs.io (free version).

**Comment by climbfuji** [comment]
Surprisingly, this fixed itself! See https://github.com/JCSDA/spack-stack/pull/805. My best guess is that adding a new workflow to `.github/workflows` must have triggered "something".

We still have the issue for `jedi-docs` in the JCSDA public repo, which uses `readthedocs.com` instead of `readthedocs.org`. I will check if the same trick works there as well, and report back.

**Comment by climbfuji** [comment]
I got this to work for jedi-docs internal, too! See https://github.com/JCSDA-internal/jedi-docs/pull/669

The one thing I want to mention here is that we build the official documentation from JCSDA/jedi-docs, and the resulting URL is https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest.

Since we create PRs for JCSDA-internal/jedi-docs, the PR (and internal develop branch) are available under https://jointcenterforsatellitedataassimilation-jedi-docs-internal.readthedocs-hosted.com/en/latest.

That's not a problem, because the PR page in jedi-docs will take you right to the correct URL (e.g. https://jointcenterforsatellitedataassimilation-jedi-docs-internal--669.com.readthedocs.build/en/669/), but when people want to access the official documentation, they'll go there anyway. Even if they mistakenly end up on the internal jedi-docs page for latest = develop (e.g. when selecting "latest" from within PR doc link), that's not a problem, since develop is always synchronized.

Closing this as completed!
